server/routes/visitJourney/dateAndTime.test.ts

```diff
@@ -91,6 +98,15 @@ const slotsList: VisitSlotList = {
             visitRoomName: 'room name',
             visitRestriction: 'OPEN',
           },
+          {
+            id: '5',
```

Comment:
There's already an `id` with the value `5` within this `slotsList` so worth renumbering to remove duplicate - otherwise we might get unexpected results